presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Fix history leak in coordinator for failed or canceled queries.
+* Fix memory leak related to query tracking in coordinator that was introduced
+  in :doc:`/release/release-0.213`.
+* Fix planning failures when lambdas are used in join filter expression.
+* Fix responses to client for certain types of errors that are encountered
+  during query creation.
+* Improve error message when an invalid comparator is provided to the
+  :func:`array_sort` function.
+* Improve performance of lookup operations on map data types.
+* Improve planning and query performance for queries with ``TINYINT``,
+  ``SMALLINT`` and ``VARBINARY`` literals.
+* Make distributed ``ORDER BY`` and aggregation unspill progress on early yield
```

Comment:
Fix issue where queries containing distributed ``ORDER BY`` and aggregation could sometimes fail to make progress when data was spilled.